### DISCUSSION TITLE
Sending docker image web traffic to stdout

### DISCUSSION THREAD

**Discussion by doodlemania2** [directus/directus#4000] [Archive]
Is it possible to send the docker image's web traffic to stdout? I'd like to capture it.

**Comment by WoLfulus** [comment]
Requests are already logged in stdout, what kind of data you're expecting to be logged?

You can always use hooks to output the information you need too.

**Reply by doodlemania2** [reply]
Hmmm - possible I'm just not grabbing them correctly then. Thanks!
